src/DynamoCoreWpf/ViewModels/Core/NoteViewModel.cs

```diff
@@ -1,16 +1,24 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Windows.Input;
```

Comment:
what is this used for?

Reply:
this is used in the `SelectNoteAndPinnedNode` method, as `SelectModelCommand` needs keyboard modifiers